src/style.css

```diff
@@ -1,3 +1,52 @@
 body {
   margin: 0;
+  display: block;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.link {
+  text-decoration: none;
+  color: black;
+  cursor: pointer;
+  padding: 20px 0 ;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.logo {
+  margin-top: 10px;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__item {
+  margin-left: 20px;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Add one more font as fallback font and move this css prop(font-family) to the very top of your block tree.
The font is always defined at the topmost classes, usually in body or html